source/requirements-dev.txt

```diff
@@ -2,7 +2,7 @@ avro==1.11.1
 wheel
 black
 boto3
-aws_cdk_lib~=2.59.0
+aws_cdk_lib>=2.59.0
```

Comment:
Could we make this version >=2.62?

Reply:
It will always install the newest version, now is 2.63.0, but I can change it to 2.62.0. Thanks!